AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -175,6 +176,15 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor
   _placeholderTextKitComponents.textView.backgroundColor = backgroundColor;
 }
 
+- (void)setTextContainerInset:(UIEdgeInsets)textContainerInset
+{
+  ASDN::MutexLocker l(_textKitLock);
+
+  _textContainerInset = textContainerInset;
+  _textKitComponents.textView.textContainerInset = textContainerInset;
```

Comment:
Thanks for the review. Would that be the case if we were accessing `self.textView`. In this case we're accessing the `textView` property from `_textKitComponents` which seems to be generated on `self.textView` instead.

Also I just accidentally nuked my master branch so I'm going to try to reopen this :)